packages/opentelemetry-types/src/metrics/Metric.ts

```diff
@@ -86,3 +86,16 @@ export interface Metric<T> {
    */
   setCallback(fn: () => void): void;
 }
+
+/**
+ * key-value pairs passed by the user.
+ */
+export type Labels = Record<string, string>;
+
+/**
+ * Canonicalized labels with an unique string labelSetKey.
+ */
+export interface LabelSet {
+  labelSetKey: string;
```

Comment:
What do you think about calling this something like `encoded`? I think we should avoid having the word "key" or "value" in this to prevent someone from thinking this is part of the actual dictionary

Reply:
I would prefer something like `identifier` that doesn't have as much implied meaning as `encoded`. To me, `encoded` implies that it is a decodable version of the LabelSet, where I don't think that is actually guaranteed by the spec. I think it's just meant to be a unique identifier.

Reply:
@dyladan is correct in that it is meant to be a unique identifier. We are using `encoded` in Python as well as Go implementation however.

Reply:
I am okay with `identifier` as well. It diverges from other SDKs, but I think it is more clear